apps/angular/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,8 @@
-import { Component, Input } from '@angular/core';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-list-item',
-  template: `
-    <div class="border border-grey-300 py-1 px-2 flex justify-between">
-      {{ name }}
-      <button (click)="delete(id)">
-        <img class="h-5" src="assets/svg/trash.svg" />
-      </button>
-    </div>
-  `,
+  template: ` <ng-content></ng-content> `,
```

Comment:
the delete button should stay inside this component, otherwise this component is totally useless.